### ISSUE TITLE
Feature Request: OPSEC Threat Modeling

### ISSUE THREAD

**Issue by PowerPress** [OWASP/threat-dragon#861]
**Describe what problem your feature request solves**:
There is no OPSEC based threat modeling unless its in context SDLC

**Describe the solution you'd like**:
I would like an option to use Threat-Dragon to create OPSEC or Operation Security based threat model

**Additional context**:
Most threat models are defined in terms of software developers life cycle or IT. OPSEC applies to IT and all domains especially physical. I could find no OPSEC threat models and think this could easily be added to Threat-Dragon and increase the user baser. The ability to easily create OPSEC threat models and display them especially for the physical domain has a lot of value. See an good example here: https://old.reddit.com/r/opsec/comments/eh6uvc/want_to_learn_opsec_as_a_total_beginner_start_here/

**Comment by jgadsden** [comment]
yes, certainly sounds interesting - can you take this on @PowerPress ?
I will assign it to you, but please decline it if you do not have the time to do it

**Comment by PowerPress** [comment]
I apologize I will have to decline the assignment

I'm not sure how to officially decline it
